lib/ufe/UsdAttribute.cpp

```diff
@@ -79,12 +80,20 @@ bool setUsdAttr(const PXR_NS::UsdAttribute& attr, const T& value)
 	return attr.Set<T>(value);
 }
 
-std::string getUsdAttributeValueAsString(const PXR_NS::UsdAttribute& attr)
+PXR_NS::UsdTimeCode getCurrentTime(const Ufe::SceneItem::Ptr& item)
+{
+	// Attributes with time samples will fail when calling Get with default time code.
+	// So we'll always use the current time when calling Get. If there are no time
+	// samples, it will fall-back to the default time code.
+	return MayaUsd::ufe::getTime(item->path());
+}
```

Comment:
New helper function that I'll call from every Get call below.

Reply:
Wouldn't this helper function be useful to have in Utils.h?

Reply:
There is already a getTime helper in Utils.h that takes a Ufe::Path. I created this one here mostly so I can have a single place that gets the time with the comment about why.

Reply:
I was coming from the angle that maybe someone else will need it as well. Anyhow, not a blocker, we can deal with it next time we will see a similar method.